### PR TITLE
updated context target 

### PR DESCRIPTION
**Pull request by bw2**
This is to work around this kubectl error:
 'The connection to the server localhost:8080 was refused - did you specify the right host or port?'

### CODE REVIEW

File: projects/gnomad/deploy/Makefile [hunk 1]
```diff
@@ -23,12 +23,12 @@ SERVICE_FORWARDING_RULE_ID=$(shell gcloud compute forwarding-rules list --format
...
-	gcloud container clusters get-credentials $(CLUSTER_NAME) --zone=$(GCLOUD_ZONE)
+	gcloud container clusters get-credentials $(CLUSTER_NAME) --zone=$(GCLOUD_ZONE) --project=$(GCLOUD_PROJECT)
 	kubectl config set-context $(CLUSTER_NAME) \
-	--cluster $(GKE_CLUSTER_NAME) \
-	--user $(GKE_CLUSTER_NAME) \
+	--cluster $(CLUSTER_NAME) \
+	--user $(CLUSTER_NAME) \
 	--namespace $(CLUSTER_NAMESPACE)
-	kubectl config use-context $(CLUSTER_NAME)
+	kubectl config use-context gke_$(GCLOUD_PROJECT)_$(GCLOUD_ZONE)_$(CLUSTER_NAME)
```
Comment (nawatts) [inline projects/gnomad/deploy/Makefile]: Was it important that the arguments to `set-context` and `use-context` matched?

Previously, this ran
```
set-context $(CLUSTER_NAME)
use-context $(CLUSTER_NAME)
```

Now it runs
```
set-context $(CLUSTER_NAME)
use-context gke_$(GCLOUD_PROJECT)_$(GCLOUD_ZONE)_$(CLUSTER_NAME)
```

**Comment by nawatts** [comment]
There's a similar `context` block in cluster/Makefile... does it need to be changed as well?

https://github.com/macarthur-lab/gnomadjs/blob/2169e0b5af52403fa77aa05dbd0266626090078c/cluster/Makefile#L31-L37

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.DS_Store
+'.DS_Store
```
Comment (nawatts) [inline .gitignore]: I think this should stay `.DS_Store`

Comment (bw2) [reply .gitignore]: oops, good point

**Pull request merged**